### ISSUE TITLE
`declaration-colon-space-before` and sass variable

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#971]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

It is normal when rule throw errors on sass variable declaration?

> Which rule, if any, is this issue related to?

e.g. `declaration-colon-space-before`

> What CSS is needed to reproduce this issue?

e.g.

```SCSS
$my-var : 0;
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "rules": {
    "declaration-colon-space-before": "never"
  }
}
```

> Which version of stylelint are you using?

e.g. `5.2.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS plugin

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SCSS

> What did you expect to happen?

Error or not?

> What actually happened (e.g. what warnings or errors you are getting)?

e.g. "The following warnings were flagged:"

```shell
test.css
Unexpected whitespace before ":" (declaration-colon-space-before) [stylelint]
```

**Comment by evilebottnawi** [comment]
Also `declaration-colon-space-after` with
```scss
$my-color:#fff;
```
